semcore/radio/src/Radio.jsx

```diff
@@ -41,7 +41,7 @@ class RadioGroupRoot extends Component {
     const { Children } = this.asProps;
 
     return (
-      <Root render={Flex} direction='column' role='radiogroup' __excludeProps={['onChange']}>
+      <Root render={Flex} direction='column' role='group' __excludeProps={['onChange']}>
```

Comment:
Do we really need it?

Reply:
It is in the task, so... I think yes

Reply:
@msereniti Yes, we need it 😎 Since we have native inputs with `type="radio"` we don't need to set `radiogroup` to them additionally. We just need to set `group` role.